.github/workflows/nodejs-ci.yml

```diff
@@ -15,6 +15,11 @@ on:
       build_command:
         type: string
         default: "npm run build"
+      run_dev_test:
+        description: "A boolen value to enable Developer tests (Unit/Integration/etc.,) are available in your repo code"
```

Comment:
Second part of the description isn't quite understandable. Can it be written better?

Reply:
updated